src/common/components/repository-row/index.js

```diff
@@ -405,9 +406,19 @@ RepositoryRowView.propTypes = {
   registerNameStatus: PropTypes.shape({
     success: PropTypes.bool
   }),
-  dispatch: PropTypes.func,
   registerNameIsOpen: PropTypes.bool,
-  configureIsOpen: PropTypes.bool
+  configureIsOpen: PropTypes.bool,
+  authActions: PropTypes.object,
+  nameActions: PropTypes.object,
+  snapActions: PropTypes.object
 };
 
-export default connect()(RepositoryRowView);
+function mapDispatchToProps(dispatch) {
+  return {
+    authActions: bindActionCreators(authStoreActionCreators, dispatch),
```

Comment:
I kind of prefer the idea of only passing that is needed by component, not all actions from given module.

But I don't mind. Lets try different patterns and come up with some conventions later on.